vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -179,7 +180,7 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal>
   std::shared_ptr<react::uwp::AppTheme> m_appTheme;
   Mso::CntPtr<react::uwp::AppearanceChangeListener> m_appearanceListener;
 #endif
-  Mso::DispatchQueue m_uiQueue;
+  winrt::com_ptr<Mso::React::ISimpleDispatch> m_uiQueue;
```

Comment:
>  winrt::com_ptr [](http://example.com/codeflow?start=0&length=16)

It wouldbe nice to use Mso::CntPtr with the Mso:: interfaces and use the Mso::UnknownObject to implement it.